src/components/List/ListItem.tsx

```diff
@@ -104,6 +133,10 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     this.setState({ isHovering: false })
   }
 
+  componentDidUpdate() {
+    this.focusableItem.focus(ReactDOM.findDOMNode(this.itemRef.current!) as HTMLElement)
+  }
```

Comment:
At first look, it's a bit confusable, as seems like it is going to be focused anyway. But inside ```this.focusableItem.focus``` method there is a check which actually decides focus or not.
Saying that, can we rename it to ```tryFocus```?

Reply:
like it, renamed